cpp/arcticdb/util/name_validation.cpp

```diff
@@ -118,8 +119,14 @@ void verify_library_path_part(const std::string& library_part, char delim) {
     }
 }
 
-void verify_library_path_on_write(const StringId& library_path) {
+void verify_library_path_on_write(const Store* store, const StringId& library_path) {
```

Comment:
@alexowens90, I am not sure how your comment about "switching from raw pointer to shared pointer" got deleted while I was editing my comment.
Anyway, I was using a using a shared pointer until this comment by Vasil. I think it would be helpful to choose a coding style across the team.